### PR TITLE
[MODULAR] Brings back Access Helpers from #13643

### PR DESCRIPTION
**Pull request by Jolly-66**
## About The Pull Request
Brings Access Helpers from #13643 because of merging shit that happened. 

## How This Contributes To The Skyrat Roleplay Experience
It doesn't (code stuff).

## Changelog

N/A

### CODE REVIEW

File: modular_skyrat/modules/skyrat_access_helpers/accesshelpers.dm [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/* Read the main access_helpers.dm for more information on these mapping tools.
+TLDR:
+Any requires EITHER OR access defined on the ID
+All requires THAT specific access defined on the ID (so don't put more than one on a door at a time UNLESS ITS INTENTIONAL for a job to have all the access(es))
+For mapping examples, look at /TG/s MetaStation
+*/
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/skyrat_access_helpers/accesshelpers.dm]: Common formatting for multi-line comments. Check out this handy extension, if you don't have it already:
https://marketplace.visualstudio.com/items?itemName=stylemistake.auto-comment-blocks
```suggestion
/** Read the main access_helpers.dm for more information on these mapping tools.
 * TLDR:
 * Any requires EITHER OR access defined on the ID
 * All requires THAT specific access defined on the ID (so don't put more than one on a door at a time UNLESS ITS INTENTIONAL for a job to have all the access(es))
 * For mapping examples, look at /TG/s MetaStation
*/
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by Gandalf2k15 — APPROVED** [review]
